clients/python-wrapper/lakefs/object_io.py

```diff
@@ -221,29 +223,65 @@ def _extract_etag_from_response(headers) -> str:
         etag = headers.get("ETag", "").strip(' "')
         return etag
 
-    def _upload(self, content, pre_sign, content_type: Optional[str] = None,
-                metadata: Optional[dict[str, str]] = None) -> ObjectStats:
-        if not pre_sign:
-            raise UnsupportedOperationException("Upload currently supported only in pre-sign mode")
-
-        headers = {}
-        if content_type is not None:
-            headers["Content-Type"] = content_type
+    def _upload_raw(self, content, content_type, metadata):
+        """
+        Preform upload api using raw api call to bypass validation of content parameter
+        """
+        headers = {
+            "Accept": self._client.sdk_client.objects_api.api_client.select_header_accept(['application/json']),
+            "Content-Type": content_type if content_type is not None else "application/octet-stream"
```

Comment:
Removed

Reply:
When providing no default we get a 500 error:
```
"message":"mime: no media type"
```
Our API also provides a default Content-Type, since our default mode is "wb", I thought it will make sense to set default Content-Type to binary